exercises/secret-handshake/instructions.md

```diff
@@ -0,0 +1,47 @@
+# Instructions
+
+Your task is to convert a number between 1 and 31 to a sequence of actions in the secret handshake.
+
+The sequence of actions are chosen by looking at the rightmost five digits of the number once it's been converted to binary.
+Start at the right-most digit and move left.
+
+The actions for each number place are:
+
+```plaintext
+00001 = wink
+00010 = double blink
+00100 = close your eyes
+01000 = jump
+10000 = Reverse the order of the operations in the secret handshake.
+```
+
+Let's use the number `9` as an example:
+
+- 9 in binary is `1001`.
+- The digit that is farthest to the right is 1, so the first action is `wink`.
+- Going left, the next digit is 0, so there is no double-blink.
+- Going left again, the next digit is 0, so you leave your eyes open.
+- Going left again, the next digit is 1, so you jump.
+
+That was the last digit, so the final code is:
+
+```plaintext
+wink, jump
+```
+
+Given the number 26, which is `11010` in binary, we get the following actions:
+
+- double blink
+- jump
+- reverse actions
+
+The secret handshake for 26 is therefore:
+
+```plaintext
+jump, double blink
+```
+
+```exercism/note
+If you aren't sure what binary is or how it works, check out [this tutorial][intro-to-binary].
```

Comment:
```suggestion
If you aren't sure what binary is or how it works, check out [this binary tutorial][intro-to-binary].
```

I am basing this recommendation on advice I've read that told me not to write link text like "this tutorial". Such places that I've seen this advice:

https://accessibility.huit.harvard.edu/technique-writing-link-text

> Bad example:
> I think you should read [this](#) article about link text.

https://developers.google.com/style/link-text

> Don't use phrases such as *this document*, *this article*, or *click here*.
> Also not recommended:
> For more information, see [this document](#).

Reply:
Good call!